packages/playground/website/builder/builder.js

```diff
@@ -0,0 +1,921 @@
+const importStartPlaygroundWeb = import('//localhost:7001/client/index.js');
+const fetchBlueprintSchema = fetch(
+	'//localhost:7001/blueprints/blueprint-schema.json'
+).then((r) => r.json());
```

Comment:
This will work in dev but not in production, [my previous comment discusses](https://github.com/WordPress/wordpress-playground/pull/773#discussion_r1412058977) a solution that would work in production and also enable others to test this PR.

Reply:
@adamziel Of course. I was just up too late editing and accidentally left this in.